8Knot/queries/company_query.py

```diff
@@ -42,82 +37,51 @@ def company_query(self, repos):
 
     query_string = f"""
                     SELECT
-                        c.cntrb_id,
-                        c.created_at AS created,
+                        left(c.cntrb_id::text, 15), -- first 15 characters of the uuid
```

Comment:
This is kind of a bad idea because those first 15 can be non-unique.

Reply:
When looking at all of our cntrb_ids the most unique numbers before the 0s start was 8